browser/extensions/api/brave_extensions_api_client.cc

```diff
@@ -9,10 +9,15 @@
 
 namespace extensions {
 
+bool IsBraveSecurityUrl(const GURL& url) {
+  return url.DomainIs("sb-ssl.brave.com") ||
+      url.DomainIs("safebrowsing.brave.com");
+}
```

Comment:
brave/brave-browser#6003 also mentions `safebrowsing2.brave.com`. Is it no longer needed?

Reply:
It's not *yet* needed as it doesn't exist yet (it will be added later in #4674).

Reply:
I don't think this is necessary because safebrowsing isn't called through the profile network context. cc @jumde